software/CONFIGURATION.md

```diff
@@ -42,7 +42,7 @@ default configuration:
 # Experimental configuration
 
 Other configuration properties are used by [experimental features](software/firmware/experimental/__init__.py)
-and can be set using a similar static configuration file. This file is located at `/config/config_ExperimentalConfig.json` 
+and can be set using a similar static configuration file. This file is located at `/config/config_ExperimentalConfig.json`
```

Comment:
> `/config/config_ExperimentalConfig.json`

Any interest in dropping one or two of the repeated "config"s here?

https://youtu.be/O5dmxBUbzBU?si=emkYK92HQjbnymUe

Reply:
Potentially, though at this point we'd be breaking a lot of contrib scripts with that change. I'm not sure the benefits outweigh the inconvenience to users if the have to reconfigure things and/or clean up junk files on their modules.

Reply:
[required] One other note. If we do remove the `config_` prefix, then the default config generator, located at `./scripts/generate_default_configs.py` will also need to change to generate the proper file names.

Reply:
I believe that script uses the `config_filename(cls)` method already, so it wull automatically use the updated filenames.

Reply:
ok, yeah, of course it does, i should have looked at the script before commenting.